cmd/environment.go

```diff
@@ -6,6 +6,29 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var envCmd = &cobra.Command{
+	Use:    "env",
+	Hidden: true,
+	Run: func(cmd *cobra.Command, _ []string) {
+		vars := map[string]string{
+			"EXOSCALE_API_KEY":         gCurrentAccount.Key,
+			"EXOSCALE_API_SECRET":      gCurrentAccount.Secret,
+			"EXOSCALE_API_ENDPOINT":    gCurrentAccount.Endpoint,
+			"EXOSCALE_API_ENVIRONMENT": gCurrentAccount.Environment,
+		}
+
+		unset, _ := cmd.Flags().GetBool("unset")
+
+		for k, v := range vars {
+			if unset {
+				fmt.Printf("unset %s\n", k)
+			} else {
+				fmt.Printf("export %s=%q\n", k, v)
```

Comment:
```suggestion
				fmt.Printf("export %s='%q'\n", k, v)
```

Reply:
@exo-cedric `%q` already encloses the value with quotes, it's equivalent to `"%q"`.